package.json

```diff
@@ -2,6 +2,7 @@
   "name": "phoenix",
   "private": true,
   "dependencies": {
+    "@opentermsarchive/engine": "^1.1.2",
```

Comment:
why is this npm package becoming a phoenix dependency?

Reply:
Hm, it seems to come from a [commit from 5 months agp](https://github.com/tosdr/edit.tosdr.org/commit/20f58909dcb9aa4a9530777bcce75f1cd6151f19), you should probably run `npm remove @opentermsarchive/engine` in the repo root and then commit that.

Reply:
Done